tests/cql_query_test.cc

```diff
@@ -4261,3 +4261,135 @@ SEASTAR_TEST_CASE(test_rf_expand) {
+                "    AND crc_check_chance = 1\n"
+                "    AND dclocal_read_repair_chance = 0.1\n"
+                "    AND default_time_to_live = 0\n"
+                "    AND gc_grace_seconds = 864000\n"
+                "    AND max_index_interval = 2048\n"
+                "    AND memtable_flush_period_in_ms = 0\n"
+                "    AND min_index_interval = 128\n"
+                "    AND read_repair_chance = 0\n"
+                "    AND speculative_retry = '99.0PERCENTILE';\n");
+    });
+}
+
+SEASTAR_TEST_CASE(test_describe_view_schema) {
+    return do_with_cql_env_thread([] (cql_test_env& e) {
+        e.execute_cql("CREATE TABLE ks.cf (\n"
+                "    pk blob,\n"
+                "    col1 blob,\n"
+                "    col2 blob,\n"
+                "    PRIMARY KEY (pk, col1))").get();
+        e.execute_cql("CREATE MATERIALIZED VIEW ks.cf_view\n"
+                "AS SELECT pk, col1 \n"
+                "FROM ks.cf\n"
+                "WHERE pk IS NOT NULL AND col1 is NOT NULL\n"
+                "PRIMARY KEY ( col1, pk)").get();
+        auto schema = e.local_db().find_schema("ks", "cf_view");
+        std::ostringstream ss;
+
+        schema->describe(ss);
+        BOOST_CHECK_EQUAL(ss.str(), "CREATE MATERIALIZED VIEW ks.cf_view AS\n"
+                "    SELECT col1, pk\n"
+                "    FROM cf\n"
+                "    WHERE pk IS NOT null AND col1 IS NOT null\n"
+                "    PRIMARY KEY (col1, pk)\n"
+                "    WITH CLUSTERING ORDER BY (pk ASC)\n"
+                "    AND bloom_filter_fp_chance = 0.01\n"
+                "    AND caching = {'keys': 'ALL','rows_per_partition': 'ALL'}\n"
+                "    AND comment = ''\n"
+                "    AND compaction = {'class': 'SizeTieredCompactionStrategy'}\n"
+                "    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}\n"
+                "    AND crc_check_chance = 1\n"
+                "    AND dclocal_read_repair_chance = 0.1\n"
+                "    AND default_time_to_live = 0\n"
+                "    AND gc_grace_seconds = 864000\n"
+                "    AND max_index_interval = 2048\n"
+                "    AND memtable_flush_period_in_ms = 0\n"
+                "    AND min_index_interval = 128\n"
+                "    AND read_repair_chance = 0\n"
+                "    AND speculative_retry = '99.0PERCENTILE';\n");
+        e.execute_cql("DROP MATERIALIZED VIEW ks.cf_view").get();
+        e.execute_cql(ss.str()).get();
+        auto new_schema = e.local_db().find_schema("ks", "cf_view");
+
+        cmp_schema(schema, new_schema);
+
+        e.execute_cql("CREATE INDEX cf_index ON ks.cf (col2);").get();
+        schema = e.local_db().find_schema("ks", "cf_index_index");
+
+        std::ostringstream ss1;
+        schema->describe(ss1);
+        BOOST_CHECK_EQUAL(ss1.str(), "CREATE INDEX cf_index ON ks.cf(col2);\n");
```

Comment:
again, shouldn't schemas be compared again with `cmp_schema` after the second change?